### DISCUSSION TITLE
Absurdly good

### DISCUSSION THREAD

**Discussion by leighton-carr** [mantinedev/mantine#5504] [Kind words]
Hope this kind of post is ok - just wanted to say thank you.

I've been writing software professionally for 25 years, with the last 15 in web (mostly internal projects in my company).  This is easily the best component library I've ever used. 

In every other instance I've run into the boundaries of what the library does and have to spend a lot of time and energy customizing or extending it.  Not only does Mantine provide easy access to everything under the hood, but 99% the things you need are provided as default options.  I've started to lose count of the "guess I'd better build my standard xyz... oh wait, they have that too" moments.  Also the docs are perfect.

Bravo, thank you, and Happy New Year.

**Comment by rtivital** [comment]
Thanks for kind words!

**Reply by MathRobin** [reply]
I approve too ! Thanks for this great work ! Bravo !

**Comment by tuler** [comment]
Docs are really good. Congrats and thank you @rtivital

**Comment by pc-gs** [comment]
Well done!! I'm using ChakraUI for personal projects, but I gonna start to use Mantine in my next project for sure.
